src/components/engagement/dto/status.enum.ts

```diff
@@ -26,6 +26,29 @@ export enum EngagementStatus {
   Rejected = 'Rejected',
 }
 
+export const TerminalEngagementStatuses: EngagementStatus[] = [
+  EngagementStatus.Terminated,
+  EngagementStatus.Completed,
+  EngagementStatus.Converted,
+  EngagementStatus.Unapproved,
```

Comment:
I'm actually wondering if Unapproved is terminal.

Reply:
@sethmcknight is `Unapproved` a terminal engagement status?

Reply:
There aren't any engagements in the db with this status, so I'm not too worried about it. Guessing it's obsolete.

Reply:
`Unapproved` is the equivalent of `DidNotDevelop`, so yes it is terminal. This status and probably some others should most likely be removed, but not in this PR.